src/pages/public_registration/action_accordions/TicketAssignee.tsx

```diff
@@ -35,12 +39,10 @@ export const TicketAssignee = ({ ticket, refetchTickets }: TicketAssigneeProps)
     onError: () => dispatch(setNotification({ message: t('feedback.error.update_ticket_assignee'), variant: 'error' })),
   });
 
-  const canSetAssignee =
-    window.location.pathname.startsWith(UrlPathTemplate.TasksDialogue) &&
-    canEditTicket &&
-    (ticket.status === 'Pending' || ticket.status === 'New');
+  const isOnTaskPage = window.location.pathname.startsWith(UrlPathTemplate.TasksDialogue);
+  const canSetAssignee = isOnTaskPage && canEditTicket && (ticket.status === 'Pending' || ticket.status === 'New');
```

Comment:
```suggestion
  const canSetAssignee = canEditTicket && (ticket.status === 'Pending' || ticket.status === 'New');
```

Kunne fjernet `isOnTaskPage` herifra, da `isOnTaskPage` brukes først i return statement, men synes det er greit å ha den her også for å tydeliggjøre.